Add Homebridge docs (#904)
MichaIng/DietPi-Docs: docs/software/home_automation.md

Official docs: <https://www.openhab.org/docs/>  
Official community: <https://community.openhab.org/>

## Homebridge

Homebridge allows you to integrate with smart home devices that do not natively support HomeKit. There are over 2,000 Homebridge plugins supporting thousands of different smart accessories.

![Homebridge logo](../assets/images/dietpi-software-homeautomation-homebridge.svg){: width="300" height="300" loading="lazy"}

=== "Access to the web interface"

    The web interface is accessible via port **8581**:

    - URL: `https://<your.IP>:8581`
    - Username/Password: Are set at first web interface login

=== "Directories"

>     - Install directory: `/usr/local/bin/hb-service`

Reviewer: This is the executable symlink. What comes closest to the install directory is `/opt/homebridge`, which also contains the actual binary.

Author: Fixed in https://github.com/MichaIng/DietPi-Docs/pull/904/commits/d7930000fca34c4b85241207c9496fc29ef68ec2

    - Config, Log & Backup folder: `/var/lib/homebridge`

=== "View logs"

    ```sh
    journalctl -u homebridge

>     cat /var/lib/homebridge/homebridge.log

Reviewer: Might be actually an idea to move all logs to journal. Is the `homebridge.log` actually already a duplicate of what is logged to stdout/journal already? At least the journal contains more than than what systemd itself logs about the service start. Might reduce some redundancy and disk I/O if we disable the plain text log file.

However, until we change something, the info here is of course correct.

Author: No, unfortunately the journal has a basic logging like this:
```
root@pi:/opt/homebridge# journalctl -u homebridge
Jul 27 00:37:35 pi systemd[1]: Starting homebridge.service - Homebridge...
Jul 27 00:37:35 pi run-parts[3733]: Ensuring /var/lib/homebridge and /opt/homebridge are owned by homebridge
Jul 27 00:37:35 pi systemd[1]: Started homebridge.service - Homebridge.
Jul 27 00:37:36 pi start.sh[3735]: [7/27/2023, 12:37:36 AM] [HB Supervisor] Homebridge Storage Path: /var/lib/homebridgeJul 27 00:37:36 pi start.sh[3735]: [7/27/2023, 12:37:36 AM] [HB Supervisor] Homebridge Config Path: /var/lib/homebridge/config.json
Jul 27 00:37:36 pi start.sh[3735]: [7/27/2023, 12:37:36 AM] [HB Supervisor] Logging to /var/lib/homebridge/homebridge.log
```

and I don't know a way to disable the logging / move the logging to the journal.

Reviewer: I cannot find a way to change this either. The official docs do not give a hint either: https://github.com/homebridge/homebridge/wiki/Homebridge-Config-JSON-Explained

I guess it is some library controlling the logs, hence with hidden config keys for it. However, not that important, we can look into it or ask at the HB community at a later time.

    ```

=== "Service handling"

    Use the following commands to control the homebridge system service:

    ```sh
    hb-service status
    ```

    ```sh
    hb-service start
    ```

    ```sh
    hb-service stop
    ```

    ```sh
    hb-service restart
    ```

=== "Update Node.js"

>     Use the following command to update the bundled Node.js runtime:
> 
>     ```sh
>     # don't use sudo if running from the Homebridge UI Terminal

Reviewer: ```suggestion
    Use the following command to update the bundled Node.js runtime (do not use `sudo` if running from the Homebridge UI Terminal):

    ```sh
```
